pkg/sentry/certs/store.go

```diff
@@ -80,7 +80,7 @@ func CredentialsExist(conf config.SentryConfig) (bool, error) {
 	return false, nil
 }
 
-/* #nosec */
+/* #nosec. */
```

Comment:
Can this be handled as an exception? This is odd.

Reply:
Sure.